tests/unit/pages/create-or-edit-lesson/CreateOrEditLesson.spec.jsx

```diff
@@ -15,4 +22,19 @@ describe('CreateOrEditLesson', () => {
     expect(titleInput).toBeInTheDocument()
     expect(descriptionInput).toBeInTheDocument()
   })
+
+  it('should add attachments', () => {
+    const addedAttachment = screen.getByText('lesson.labels.attachments')
+    expect(addedAttachment).toBeInTheDocument()
+
+    fireEvent.click(screen.getByText('lesson.labels.attachments'))
+  })
+
+  it('display validation error if title or description is empty', () => {
+    const titleInput = screen.getByLabelText('lesson.labels.title')
+    fireEvent.change(titleInput, { target: { value: '' } })
+
+    const descriptionInput = screen.getByLabelText('lesson.labels.description')
+    fireEvent.change(descriptionInput, { target: { value: '' } })
```

Comment:
same as above, you should get errors by text and check if they are in the document. 
Also, add spaces between variable declaration, fireEvent and expects in your tests